### ISSUE TITLE
@inheritdoc for static getters (properties)

### ISSUE THREAD

**Issue by damirka** [jsdoc3/jsdoc#1229]
Here is the case:

```JavaScript
class Brand {
  /**
   * Car brand name
   *
   * @static
   * @readonly
   * @type {string}
   */
  static get brandName () {
    return '';
  }
}

class Hennessy extends Brand {
  /**
   * @inheritdoc
   */
  static get brandName () {
    return 'Hennessy';
  }
}
```

No description for Hennessy.brandName after docs generation.
What am I doing wrong?

**Comment by m-turek** [comment]
Hi, any update on this?

**Comment by damirka** [comment]
@m-turek nah, no updates. Gave up finding the solution. Tried [ESDoc](https://github.com/esdoc/esdoc) as it's made for class-related things, but I've opened multiple issues there which make ESDoc even worse solution than JSDoc when working with pure JS.

**Comment by Reinmar** [comment]
We decided that the easiest way to workaround this is by writing a JSDoc plugin which fixes the inheritance of static members. It's released in https://www.npmjs.com/package/@ckeditor/jsdoc-plugins (with other workarounds and improvements) and can be used like this:

https://github.com/ckeditor/ckeditor5-dev/blob/c330cfa7309ec2e08e419d733f7271262d4041b4/packages/ckeditor5-dev-docs/lib/index.js#L39

And it works for us like a charm:

![image](https://cloud.githubusercontent.com/assets/156149/26199542/b96ea14c-3bca-11e7-8dfb-2ea79ce61651.png)

**Comment by mercmobily** [comment]
I just wanted to chime in and say that this would help me a lot too

**Comment by LukasWillin** [comment]
Would be really great if that could be done.
